cpp/src/io/parquet/reader_impl.cu

```diff
@@ -464,8 +464,9 @@ class aggregate_metadata {
    *
    * @param names List of column names to load, where index column name(s) will be added
    */
-  void add_pandas_index_names(std::vector<std::string>& names) const
+  std::vector<std::string> get_pandas_index_names() const
```

Comment:
The mention of Pandas in a `cu` file makes me die a little on the inside. :grimacing:

Reply:
Yeah? how about pandas example code in a cu file? https://github.com/rapidsai/cudf/blob/7721819eeed68115fd4d7033cba016830b0afcd8/cpp/src/strings/split/split.cu#L382-L412

Reply:
:(